v2/pkg/genruntime/root_resources.go

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package genruntime
+
+// TODO: The generated types should impl this
```

Comment:
Make an issue for this so we track it and come back to complete.

Reply:
Or is this done already?

Reply:
It's currently manually implemented on the one location-based resource we have (which is `ResourceGroup`)

Reply:
but that does mean that there isn't really anything to do right now, as no resources actually require to to be automatically generated onto them

Reply:
Filed #2420